### PR TITLE
Fixed misspelled variable name.

### PR DESCRIPTION
**Pull request by Jean-PhilippeGoerke**
Fix for compiling windows version.

### CODE REVIEW

File: src/arvnetwork.c [hunk 1]
```diff
@@ -224,7 +224,7 @@ arv_enumerate_network_interfaces (void)
 				broadaddr->sin_addr.s_addr |= ~(mask->sin_addr.s_addr);
 			}
 			#if WINVER >= _WIN32_WINNT_VISTA
-			else if (lpSocketaddr->sa_family == AF_INET6){
+			else if (lpSockaddr->sa_family == AF_INET6){
 				arv_warning_interface("IPv6 support not yet implemented.");
 			}
 			#endif
```
Comment (kalvdans) [inline src/arvnetwork.c:230]: Remove the conditional compilation to make such misspellings discovered earlier:
```
}
else if (WINVER >= _WIN32_WINNT_VISTA && lpSockaddr->sa_family == AF_INET6){
   ...
}
```

**Comment by Jean-PhilippeGoerke** [comment]
> @Jean-PhilippeGoerke , would you mind to describe your way to build aravis on Windows ?

Hi @EmmanuelP,
I am currently using MSYS2 environment in which I have installed MINGW and meson.
Within MinGW 64 I follow the instructions on the github site to use meson for configuration and compilation.
I don't know if this is the best way to do it as I am ususally working with VisualStudio only, but compilation works and the dll can be used without any issue so far.

Jean-Philippe

**Comment by eudoxos** [comment]
> If I'm not mistaken, @eudoxos, you are cross compiling, am I right ?
Corrent, I was cross-compiling.

**Comment by eudoxos** [comment]
> I am currently using MSYS2 environment in which I have installed MINGW and meson.
> Within MinGW 64 I follow the instructions on the github site to use meson for configuration and compilation.
This is good, that means creating a MSYS package will be easy (that has to be done in real windows; msys2 currently fails to run under wine AFAIK)

File: src/arvnetwork.c (hunk 1 above)
Comment (kalvdans) [reply src/arvnetwork.c:230]: The vista workarounds are a mess anyway... better not touch with a stick.

**Pull request merged**